pkg/plugin/state.go

```diff
@@ -728,16 +738,33 @@ func (e *AutoscaleEnforcer) reserveResources(
 	}
 	newNodeReservedCPU := node.cpu.Reserved + ps.cpu.Reserved
 	newNodeReservedMem := node.mem.Reserved + ps.mem.Reserved
+	newNodeBufferCPU := node.cpu.Buffer + ps.cpu.Buffer
```

Comment:
This looks like `node.*.Buffer` is only increasing. Or am I missing something?

Reply:
During reserve, yeah, that's correct.

Reply:
How do they decrease? I guess the answer is the same as for the above comment.